test/inputs_from_gwis_config.jl

```diff
@@ -0,0 +1,152 @@
+module TestGwisEstimands
+
+using Test
+using SnpArrays
+using TargeneCore
+using Arrow
+using DataFrames
+using Serialization
+using TMLE
+using CSV
+using YAML
+
+TESTDIR = joinpath(pkgdir(TargeneCore), "test")
+
+include(joinpath(TESTDIR, "testutils.jl"))
+
+function get_summary_stats(estimands)
+    outcomes = [TargeneCore.get_outcome(Ψ) for Ψ in estimands]
+    results = DataFrame(ESTIMAND = estimands, OUTCOME = outcomes)
+    return sort(combine(groupby(results, :OUTCOME), nrow), :OUTCOME)
+end
+
+function check_estimands_levels_interactions(estimands)
+    string_treatments = YAML.load_file(joinpath(TESTDIR, "data", "config_gwis_first_order.yaml"))["extra_treatments"]
+    extra_treatments = Set(Symbol.(string_treatments))
+
+    for Ψ in estimands
+        # Get the treatment keys from the first arg
+        treatment_set = Set(keys(Ψ.args[1].treatment_values))
+        
+        # Find the variant (treatment that's not in extra_treatments)
+        variants = setdiff(treatment_set, extra_treatments)
+        
+        # Skip if no variant found (shouldn't happen, but defensive)
+        isempty(variants) && continue
```

Comment:
No this should be a test, e.g., tests are your defense against bug so you need to catch them:

```
@test length(variants) == 1
```